plugins/outputs/cloudwatch/util.go

```diff
@@ -50,19 +47,12 @@ const (
 )
 
 func publishJitter(publishInterval time.Duration) (publishJitter time.Duration) {
-	hostName, _ := os.Hostname()
-	jitter := computeMD5Hash(hostName, int64(publishInterval.Seconds()))
+	r := rand.New(rand.NewSource(time.Now().Unix()))
+	jitter := r.Int63n(int64(publishInterval.Seconds()))
```

Comment:
looking at this blog post - https://golang.cafe/blog/golang-random-number-generator.html

You should be able to just call rand.Int63n(...) and the correct thing should happen. Without seeding the RNG, you should get the system's default random number generation (/dev/urandom for most)

Reply:
I'm still looking into this. I'm not sure if the above statement is true.

Reply:
As stated in the articled you linked:
> however if you try to run it several times you will see always the same number in output
Without seeding, the same binary would always generate the same sequence of random numbers, which when a large fleet start, will deterministically create a request spike at the same time.